swift_static_framework/swift_static_framework.bzl

```diff
@@ -33,9 +33,24 @@ def _zip_generated_objc_hdr_arg(module_name, generated_objc_hdr_file):
         file_path = generated_objc_hdr_file.path,
     )
 
+def _zip_modulemap_arg(module_name, modulemap_file):
+    return "{module_name}.framework/Modules/module.modulemap={file_path}".format(
+        module_name = module_name,
+        file_path = modulemap_file.path,
+    )
+
+def _modulemap_file_content(module_name):
+    return """\
+framework module {module_name} {{
+  header {module_name}-Swift.h
```

Comment:
@thii Looks like we need to add double quotes around the header, e.g.:

framework module MyModule {
  header **"**MyModule-Swift.h**"**
  requires objc
}

Otherwise, module imports appears to work as expected!

Reply:
Good catch! Thanks.